### PR TITLE
Switch into interactive Logo

### LINKED ISSUE #38: Make logo eyes follow cursor
Please I need that https://codepen.io/loktar00/pen/FmkAd

### PR DESCRIPTION
**Pull request by elrumordelaluz**
resolve #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h1 align="center">
-  <a href="https://mythbusters.js.org"><img src="docs/logo.svg"></a>
+  <a href="https://mythbusters.js.org"><img src="https://cdn.rawgit.com/elrumordelaluz/js-mythbusters/46efb87d/docs/logo.svg"></a>
```
Comment (elrumordelaluz) [inline README.md]: I think could be better to update, once merged from the `main` repo and right `commit`.

Comment (Kikobeats) [inline README.md]: you're right

**Comment by Kikobeats** [comment]
@elrumordelaluz  merged, thanks!

One question: Have sense blink animation is different in css and in svg?

**Comment by elrumordelaluz** [comment]
Cool!

> Have sense blink animation is different in css and in svg?

I did it to have a _lightweight_ `.svg` file that you could be used as an external file, just simulated a _tiny_ animation from the interaction in _Logo_ in [coverpage](https://github.com/Kikobeats/js-mythbusters/blob/master/docs/_coverpage.md)

**Comment by Kikobeats** [comment]
awesome!

How could be possible blink more frequently but the blink animation more slowly? 😂

**Comment by elrumordelaluz** [comment]
I copied [here](https://codepen.io/elrumordelaluz/pen/dWQxpg?editors=1000) the `svg` code and added another _blink_ in the animation _cycle_. We can add more in case…

**Pull request merged**